modules/extensions-support/src/main/java/org/mule/module/extension/internal/runtime/DefaultExecutionMediator.java

```diff
@@ -126,6 +130,16 @@ private Object executeWithInterceptors(OperationExecutor executor,
         return result;
     }
 
+    private Exception processException(ExceptionEnricher exceptionEnricher, Exception e)
+    {
+        Throwable root = ExceptionUtils.getRootCause(e);
```

Comment:
this can be null. Look at the original logic at OperationMessageProcessor. This is not equivalent